CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Solidus (not yet versioned)
+
+*   Address is immutable (Address#readonly? is always true)
+
+    This allows us to minimize cloning addresses, while still ensuring historical
+    data is preserved.
+
+*   PUT /api/addresses without an order no longer works
```

Comment:
This turned out to already be the case in Solidus right? (As opposed to bonobos/spree/2-2-dev)